cli-tests/tests/output-dir.test.ts

```diff
@@ -107,22 +123,16 @@ describe("Set of --output-dir tests", () => {
 
       it("--output-dir output is presented", () => {
         expect(result.output).toMatch(/(Permission denied|Access is denied)/i);
+        tmpDirZkSolc.removeCallback();
       });
 
       // Exit code should be the same
-      xit("solc exit code == zksolc exit code", () => {
-        const solcResult = executeCommand(solcCommand, args);
-        expect(solcResult.exitCode).toBe(result.exitCode);
-
-      });
-      changeDirectoryPermissions(paths.pathToReadOnlyOutputDir, 'a');
-      removeDirectory(paths.pathToReadOnlyOutputDir);
+      // xit("solc exit code == zksolc exit code", () => {
```

Comment:
Please, remove comments if unnecessary.

Reply:
This stub should be uncommented after the fix in the zksolc output. The teset itself should contain this code.

Reply:
https://github.com/matter-labs/era-compiler-solidity/pull/54#discussion_r1532457805